### PR TITLE
feat: add WorkspaceGroupUserDeleteModal & deprecate PTableCheckModal

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
image

### CODE REVIEW

File: apps/web/src/services/advanced/components/WorkspaceGroupWokrspaceDeleteModal.vue [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+    try {
+        if (state.isRemoveSingleWorkspaceType) {
+            if (!workspaceGroupPageState.modalAdditionalData.selectedWorkspace?.workspace_id) throw new Error('Invalid workspace id');
+            await SpaceConnector.clientV2.identity.workspace.changeWorkspaceGroup<WorkspaceChangeWorkspaceGroupParameters, WorkspaceModel>({
+                workspace_id: workspaceGroupPageState.modalAdditionalData.selectedWorkspace?.workspace_id,
+            });
+        } else {
+        // eslint-disable-next-line max-len
+            await Promise.allSettled(workspaceGroupPageGetters.selectedWorkspaceIds.map((item) => SpaceConnector.clientV2.identity.workspace.changeWorkspaceGroup<WorkspaceChangeWorkspaceGroupParameters, WorkspaceModel>({
```
Comment (yuda110) [inline apps/web/src/services/advanced/components/WorkspaceGroupWokrspaceDeleteModal.vue:90]: suggestion: How about to create 'fetcher' variable for readable code?!🧐

File: apps/web/src/services/advanced/components/WorkspaceGroupWokrspaceDeleteModal.vue [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    workspaceGroupPageStore.closeModal();
+};
+
+const handleCloseModal = () => {
+    workspaceGroupPageStore.closeModal();
+    workspaceGroupPageStore.resetSelectedWorkspace();
+};
+
+const handleChangeSort = (sortBy:string, sortDesc:boolean) => {
+    console.log('>>', sortBy, sortDesc);
```
Comment (yuda110) [inline apps/web/src/services/advanced/components/WorkspaceGroupWokrspaceDeleteModal.vue:117]: here!

**Review by yuda110 — APPROVED** [review]
LGTM~!!!

File: apps/web/src/services/advanced/components/WorkspaceGroupWokrspaceDeleteModal.vue (hunk 2 above)
Comment (sulmoJ) [reply apps/web/src/services/advanced/components/WorkspaceGroupWokrspaceDeleteModal.vue:117]: 홀리

**Pull request merged**